third_party/move/move-compiler-v2/legacy-move-compiler/src/shared/mod.rs

```diff
@@ -705,6 +706,21 @@ pub mod known_attributes {
         ModuleLock,
     }
 
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
For what do we need this? Coulld not find usage

Reply:
this is used in `KnownAttribute` above.